### PR TITLE
Add testing of case sensitivity.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge or @geirst PR

### CODE REVIEW

**Review by toregge — APPROVED** [review]

File: tests/search/attributesearch/attributesearch.rb [hunk 1]
```diff
@@ -56,12 +56,18 @@ def a_r_w_e(word, resnam, explain)
     assert_result("query=fstitle:#{word}", selfdir + "case.#{resnam}.result", nil, nil, 0, explain)
   end
 
-  def test_attributesearch_case
-    deploy_app(SearchApp.new.sd(selfdir+"cased.sd"))
+  # helper method
+  def expect_fail(word, explain)
```
Comment (geirst) [inline tests/search/attributesearch/attributesearch.rb:60]: The explain parameter is not used.

File: tests/search/attributesearch/cased/cased.sd [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline tests/search/attributesearch/cased/cased.sd:1]: Please update copyright.

File: tests/search/attributesearch/attributesearch.rb [hunk 3]
```diff
@@ -56,12 +56,18 @@ def a_r_w_e(word, resnam, explain)
...
+    assert_hitcount("query=title:#{word}", 0)
+    assert_hitcount("query=fstitle:#{word}", 0)
+  end
+
+  def test_attributesearch_uncased
+    deploy_app(SearchApp.new.sd(selfdir+"uncased/cased.sd"))
```
Comment (geirst) [inline tests/search/attributesearch/attributesearch.rb:66]: Consider renaming the sd-file to _test.sd_ as this naming could be misleading.

File: tests/search/attributesearch/cased/cased.sd [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+search cased {
```
Comment (geirst) [inline tests/search/attributesearch/cased/cased.sd:3]: Consider renaming to _test.sd_. See comment above.

Comment (baldersheim) [reply tests/search/attributesearch/cased/cased.sd:3]: Fixed

File: tests/search/attributesearch/cased/cased.sd (hunk 2 above)
Comment (baldersheim) [reply tests/search/attributesearch/cased/cased.sd:1]: Fixed

File: tests/search/attributesearch/attributesearch.rb (hunk 3 above)
Comment (baldersheim) [reply tests/search/attributesearch/attributesearch.rb:66]: Fixed

File: tests/search/attributesearch/attributesearch.rb (hunk 1 above)
Comment (baldersheim) [reply tests/search/attributesearch/attributesearch.rb:60]: I am keeping it as it is documentation on the call site similar to a_r_w_e usage.
Added a comment why it is unused.

**Pull request merged**